### PR TITLE
New package: wluma-1.2.2

### PR DESCRIPTION
**Pull request by raninninn**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by biopsin** [comment]
You need to squash into one commit and change revision to one.

File: srcpkgs/wluma/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=3
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-makefile
+make_cmd="make build"
```
Comment (ericonr) [inline srcpkgs/wluma/template:9]: What you want is `make_build_target=build`

File: srcpkgs/wluma/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=3
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
```
Comment (ericonr) [inline srcpkgs/wluma/template:7]: remove commented out fields

File: srcpkgs/wluma/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="pkg-config meson ninja"
+makedepends="Vulkan-Headers wayland-devel Vulkan-Tools"
+depends="wayland vulkan-loader mesa-vulkan-intel mesa-vulkan-radeon"
+short_desc="Automatic screen brightness based on ambient light"
+maintainer="raninninn <raninninn-devel@protonmail.com>"
+license="MIT"
+homepage="https://github.com/maximbaz/wluma"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=dabd6cdf0b9c644af52ad6fad40861829f43c534d42123af2f1f4465b31d5bed
+make_check=no	# wluma provides no check suite
```
Comment (ericonr) [inline srcpkgs/wluma/template:19]:
```suggestion
make_check=no	# wluma provides no test suite
```

File: srcpkgs/wluma/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+version=1.2.2
+revision=3
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-makefile
+make_cmd="make build"
+hostmakedepends="pkg-config meson ninja"
+makedepends="Vulkan-Headers wayland-devel Vulkan-Tools"
+depends="wayland vulkan-loader mesa-vulkan-intel mesa-vulkan-radeon"
```
Comment (ericonr) [inline srcpkgs/wluma/template:12]: `wayland` and `vulkan-loader` are detected automatically:

```
=> wluma-1.2.2_3: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libvulkan.so.1 <-> vulkan-loader>=1.0.57.0_1
   SONAME: libm.so.6 <-> glibc>=2.32_1
   SONAME: libwayland-client.so.0 <-> wayland>=1.10.0_1
   SONAME: libc.so.6 <-> glibc>=2.32_1
```

And I don't think we should force the vulkan drivers either, so I'd be against them. In other words, you can completely remove `depends`.

File: srcpkgs/wluma/template [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'wluma'
+pkgname=wluma
+version=1.2.2
+revision=3
+archs="x86_64"
```
Comment (ericonr) [inline srcpkgs/wluma/template:5]: No reason to limit this.

**Comment by ericonr** [comment]
Well, I now see this uses meson, so you should probably use that build style directly.

Also, they are rewriting in Rust, so the direction of the package is quite open, I don't think it should be packaged just yet.

**Comment by raninninn** [comment]
Okay, thanks for all the help and advice.

**Pull request closed without merging**